app/views/group_offer_categories/index.html.slim

```diff
@@ -0,0 +1,21 @@
+.row
+  .col-xs-12
+    h1= t_attr(:group_offer_categories)
+
+= form_navigation_btn :new
+
+.row
```

Comment:
If this table should be enclosed in a div it should be `.table-responsive` and not row.